frontend_lib/src/component/Input/SelectStatus/SelectStatus.styl

```diff
@@ -1,6 +1,7 @@
 @import '../../../css/Variable.styl'
 
 .selectStatus
+  margin-right 15px
```

Comment:
the positioning of a generic component shouldn't be in the component itself.

The reason is that if we use this component somewhere where we must not have a margin-right: 15px, we will have to either add a props to the component to make this margin conditional or give the wrapper a negative margin to cancel it.

So, here, it's better to make each usage of the component a margin-right.

in conclusion: revert your change and add the missing margin-right to the apps that needs it